docs/fast_data/runtime_management.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: runtime_management
+title: Runtime Management
+sidebar_label: Runtime Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+
+# Runtime Management
+
+Welcome to Runtime Management - your tool for achieving better control over the Fast Data environment during runtime.
+This platform empowers you to dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices effortlessly.
+Dive into the world of streamlined runtime control and take charge of your Fast Data deployment with ease.
+
+## Overview
+
+Runtime Management is specifically designed to facilitate the runtime deployment of Fast Data.
+It provides a user-friendly web interface and robust backend functionalities to enhance your control over the Fast Data microservices environment.
+
+With Runtime Management, users gain the ability to govern potentially every aspect of the Fast Data runtime.
```

Comment:
```suggestion
With Runtime Management, users gain the ability to govern several different aspects of the Fast Data runtime configuration.
```

Reply:
Ok for `several different aspects`.

I wouldn't say "Fast Data runtime configuration" though, since it's not the configuration that we can handle with this tool, but quite Fast Data runtime behaviour.

What do you think?

Reply:
it seems to me like a repetition of the previous paragraph. Either we merge the two or we rephrase one of the two

Reply:
I have refactor the two pharagraphs. Now it's:

# Runtime Management

Runtime Management is a tool in the Fast Data environment for achieving better control over the Fast Data microservices during runtime.
This component lets you dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices with few clicks.

To facilitate these operations, it provides a user-friendly web interface and robust backend functionalities.

With Runtime Management, users gain the ability to govern several different aspects of the Fast Data runtime.

:::note
Currently, this mechanism is supported only in the Single View Creator.
:::